.circleci/config.yml

```diff
@@ -38,9 +38,31 @@ defaults: &defaults
 
 version: 2
 jobs:
+    configure_env_vars:
+        <<: *defaults
+
+        docker:
+            - image: quay.io/pantheon-public/build-tools-ci:visual-regression-vars
+
+        steps:
+            - checkout
+
+            - run:
+                # Set TERMINUS_ENV and related environment variables.
+                # https://github.com/pantheon-systems/docker-build-tools-ci/blob/4.x/scripts/set-environment
+                name: environment
+                command: /build-tools-ci/scripts/set-environment
```

Comment:
I'm still wary about running this script here. I think future enhancements are likely to cause frequent build failures here, as `set-environment` assumes that it is being run in `build-tools-ci` (since it is part of the build-tools-ci container). Running a script from a different container without using that container just does not seem like a good practice.

Let's try caching $BASH_ENV in an earlier stage and restoring + sourcing it here. Identify what other things from that script are necessary, and make a separate script to do just what is necessary here.